8BallApp/Presentation/ViewModels/BallViewModel.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  BallViewModel.swift
+//  8BallApp
+//
+//  Created by Andrii Shkliaruk on 22.02.2022.
+//
+
+import Foundation
+
+class BallViewModel {
+    let callToShakeText = Constants.callToShakeText
+    let settingsIconName = Constants.gearIconName
+    let motionBeganText = Constants.motionBeganText
+    let loadingText = Constants.loadingText
+    
+    private let networkService = NetworkService()
```

Comment:
Сервис нужно закрыть протоколом NetworkDataProvider и инжектить его через инит. Т.е. переменная тут будет
```suggestion
    private let networkDataProvider = NetworkDataProvider
```
 при инициализации вью модели передашь Нетворк сервис
`BallViewModel(networkDataProvider: NetworkService())`